### ISSUE TITLE
The classifier for Linear Evaluation Accuracy

### ISSUE THREAD

**Issue by WenjinW** [DonkeyShot21/cassle#9]
Hi,

This is an exciting and enlightening work.

I am confused by the number of classifiers for Linear Evaluation Accuracy. 

In the paper, you said, "For class-incremental and data-incremental, we use the task-agnostic setting, meaning that at evaluation time we do not assume to know the task ID". As I understand it, this means that you only maintain one classifier and continuously optimize it after learning each task for linear evaluation accuracy.

However, I found in #1 that you said, "as we operate in the class-incremental setting we train one linear classifier per task."

I would appreciate a clearer explanation.

Thanks.

**Comment by DonkeyShot21** [comment]
Hi,

> "as we operate in the class-incremental setting we train one linear classifier per task." 

This refers to the calculation of forgetting (read the whole thread for context), and was meant to correct one of the statements in the previous messages: 

> To report the forgetting, you train a set of linear classifiers for each task during the continual learning process

It is not a set of linear classifiers, it is a single classifier that discriminates all classes in the dataset. This process is repeated after / during every pre-training step (task).
